site/app/templates/grading/simple/Display.twig

```diff
@@ -109,18 +98,36 @@
         </select>
     </div>
     {% if action == 'numeric' and core.getUser().accessAdmin() %}
-        <div class="column-wrapper">
-            <label for="csvUpload">Upload CSV
-                <input class="csvButtonUpload" type="file" id="csvUpload" accept=".csv, .txt">
-            </label>
-            <p>
-                The CSV file should be formatted as such:
-                <br />
-                user id,first name,last name,grade1,grade2,...,total points earned,text1,text2,...
-            </p>
-        </div>
+		<p style="margin-bottom: 1vh">
```

Comment:
In general, using `vh` units for this type of thing is challenging to work with in the future.  It would be better to use `em`, `rem`, or `px` to accomplish this task unless I'm misunderstanding why `vh` was chosen.  Also, this is another thing which could be moved to an external stylesheet.